pennylane/liealg/__init__.py

```diff
@@ -28,6 +28,43 @@
     ~lie_closure
     ~structure_constants
     ~center
+    ~cartan_decomp
+    ~check_cartan_decomp
+    ~check_commutation
+
+Involutions
+~~~~~~~~~~~
+
+A map :math:`\theta: \mathfrak{g} \rightarrow \mathfrak{g}` from the Lie algebra :math:`\mathfrak{g}` to itself is called an involution
+if it fulfills :math:`\theta(\theta(g)) = g \ \forall g \in \mathfrak{g}` and is compatible with commutators,
+:math:`[\theta(g), \theta(g')]=\theta([g, g']).` Involutions are used to construct a :func:`~cartan_decomp`. There are seven canonical
+Cartan involutions of classical real simple Lie algebras (``AI, AII, AIII, BDI, DIII, CI, CII``)
+and one canonical involution for each real semisimple Lie algebra made up of two isomorphic
+classical simple components (``A, BD, C``).
+See, for example, Tab. 4 in `Edelman and Jeong <https://arxiv.org/abs/2104.08669>`__
+Note that the functions implemented here do not represent the mathematical involutions directly,
+but return a boolean value that indicates whether or not the input is in the :math:`+1` eigenspace
+of :math:`\theta`. When using them, it is usually assumed that we apply them to operators in the
+eigenbasis of the underlying involution :math:`\theta`.
+
+.. currentmodule:: pennylane.liealg
+
+.. autosummary::
+    :toctree: api
+
+    ~even_odd_involution
+    ~concurrence_involution
+    ~A
+    ~AI
+    ~AII
+    ~AIII
+    ~BD
+    ~BDI
+    ~DIII
```

Comment:
is DIII intentionall coming before CI CII @dwierichs

Reply:
It's a bit annoying in any case, but I think personally I shifted to liking the grouping more like I did it here, because now it is 
`A..`: "unitary stuff"
`BD..` and `D...`: "orthogonal stuff"
`C...`: "symplectic stuff"

Reply:
It's also the one used [here](https://arxiv.org/pdf/2104.08669) and by Wikipedia, for example :) 
![image](https://github.com/user-attachments/assets/ca651a46-1646-40ac-994e-bb4ad5ed0bff)

Reply:
perfect 👍

Reply:
just wanted to make sure it was intentional :)